internal/daemon/vendor-specific-plugins/marvell/main.go

```diff
@@ -672,7 +673,7 @@ func (vsp *mrvlVspServer) configureIP(dpuMode bool) (pb.IpPort, error) {
 		return pb.IpPort{}, err
 	}
 	klog.Infof("Interface Name: %s", ifName)
-	err = enableIPV6LinkLocal(ifName, addr)
+	err = vspnetutils.EnableIPV6LinkLocal(vsp.fs, ifName, addr)
```

Comment:
I like the abstraction but I'm wondering if the vspnetutils is the right abstraction here. It seems that it contains multiple smaller useful things, almost as if we might want to abstract different partitions of vspnetutils. Don't necessarily change it, but some discussion is needed as this is fundamental and at and architectural level. Again, might be that we call it a day today, and add it to the backlog to split it up if we see the benefit. It's about determining the right names to express the intent.

Reply:
Hmmm, When it grows larger, we can move it. If the daemon needs these functions that we move it to a more common place. But I think VSP is appropriate right now with the current architecture.

Reply:
Ok